crates/re_viewer/src/ui/space_view.rs

```diff
@@ -148,19 +145,24 @@ impl SpaceView {
         }
     }
 
-    pub fn selection_ui(&mut self, ctx: &mut ViewerContext<'_>, ui: &mut egui::Ui) {
+    pub fn selection_ui(
```

Comment:
I think in the near future (not now!) we want to invert some of these methods to be memberfunction of the spaceviewstate taking a blueprint state. In my thinking the blueprint state, while describing the ui, doesn't know about specifics of the ui implementation whereas the state is allowed to :)